modules/porous_flow/paper.md

```diff
@@ -23,28 +23,72 @@ date: 13 February 2020
+skeleton (so the second term ensures mass conservation under an evolving skeleton); $\mathbf{F}$ is the fluid flux vector (that includes advection by Darcy's law as well as diffusion and hydrodynamic dispersion); $\Lambda$ is a radioactive decay rate; $\phi I_{\mathrm{chem}}$ represents chemical precipitation or dissolution ($\phi$ is the porosity); and $q$ is a source.
+
+Heat-energy conservation reads
+\begin{equation}
+0 = \frac{\partial\mathcal{E}}{\partial t} + \mathcal{E}\nabla\cdot{\mathbf
+  v}_{s} + \nabla\cdot \mathbf{F}^{T} -
+\nu
+  (1-\phi)\sigma^{\mathrm{eff}}_{ij}\frac{\partial}{\partial
+    t}\epsilon_{ij}^{\mathrm{plastic}}
+ - q^{T} \ .
+\end{equation}
+Here $\mathcal{E}$ is the heat energy per unit volume in the rock-fluid
+system (which involves the porous skeleton's heat energy as well as the fluids' heat energy, accounting for their saturations); $\mathbf{F}^{T}$ is the heat flux (which involves heat conduction and advection by the fluid); $\nu$ describes the ratio of
+plastic-deformation energy that gets transferred to heat energy during plastic deformation of the solid skeleton,
+$\sigma^{\mathrm{eff}}_{ij}$ is the effective stress, $\epsilon_{ij}^{\mathrm{plastic}}$ is the plastic strain, and $q^{T}$ is a heat source.
+
+Conservation of momentum reads
+\begin{equation}
+\rho_{\mathrm{mat}}\frac{\partial v_{s}^{j}}{\partial t} =
+\nabla_{i}\sigma_{ij}^{\mathrm{eff}} - \alpha_{B}\nabla_{j} P_{f} + \rho_{\mathrm{mat}}b_{j} \ .
+\end{equation}
+Here $i$ and $j$ are spatial indices; $v_{s}^{j}$ is the $j^{\mathrm{th}}$ component of the velocity $\mathbf{v}_{s}$; $\rho_{\mathrm{mat}}$ is the
+mass-density of the material (including solid and fluid contributions); and $b_{j}$ are the
+components of the external force density (for example, the
+gravitational acceleration).  The effective stress $\sigma^{\mathrm{eff}}$ obeys a constitutive equation, $\alpha_{B}$ is the Biot coefficient, and $P_{f}$ an effective fluid porepressure.
+
+A comprehensive listing of equations, including fluid equations of state, the evolution of porosity and permeability, relative-permeability functions, capillary relationships, chemical reactions, etc, may be found at https://mooseframework.org/modules/porous_flow/index.html.
+
+# Software details and comparisons
+
+``PorousFlow`` is built upon the open-source, massively parallel, fully implicit multiphysics simulation framework MOOSE (Multiphysics Object-Oriented Simulation Environment) [@permann2020moose].  MOOSE is an open-source library from Idaho National Laboratory that provides a high-level interface to the libMesh finite element library [@libmesh] and PETSc nonlinear solvers [@petsc].  MOOSE and ``PorousFlow`` follow [strict quality controls](https://mooseframework.org/sqa/index.html).
+
 Other open-source codes that can also perform tightly-coupled THMC simulations to some degree include OpenGeoSys [@opengeosys], PFLOTRAN [@pflotran] and work under the Open Porous Media initiative [@opm].
```

Comment:
I know what you meant about not underselling or denigrating the other capable codes here. Perhaps we could add something along the lines of `One aspect that makes PorousFlow different to these codes is the ease of coupling additional physics to the base flow and transport capabilities, and the relative speed and simplicity of developing new capability using the provided APIs` or something along those lines.

Reply:
Thanks, i've added that and slightly modified the paragraph following it.